pkg/apis/imageregistry/v1/types.go

```diff
@@ -10,14 +10,55 @@ import (
 )
 
 const (
-	OperatorStatusTypeRemoved             = "Removed"
-	ImageRegistryName                     = "image-registry"
-	ImageRegistryResourceName             = "instance"
-	ImageRegistryCertificatesName         = ImageRegistryName + "-certificates"
-	ImageRegistryPrivateConfiguration     = ImageRegistryName + "-private-configuration"
+
+	// ImageRegistryName is the name of the image-registry workload resource (deployment)
+	ImageRegistryName = "image-registry"
+
+	// ImageRegistryResourceName is the name of the image registry config instance
+	ImageRegistryResourceName = "instance"
+
+	// ImageRegistryCertificatesName is the name of the configmap that is managed by the
+	// registry operator and mounted into the registry pod, to provide additional
+	// CAs to be trusted during image pullthrough
+	ImageRegistryCertificatesName = ImageRegistryName + "-certificates"
+
+	// ImageRegistryPrivateConfiguration is the name of a secret that is managed by the
+	// registry operator and which provides credentials to the registry for things like
+	// accessing S3 storage
+	ImageRegistryPrivateConfiguration = ImageRegistryName + "-private-configuration"
+
+	// ImageRegistryPrivateConfigurationUser is the name of a secret that is managed by
+	// the administrator and which provides credentials to the registry for things like
+	// accessing S3 storage.  This content takes precedence over content the operator
+	// automatically pulls from other locations, and it is merged into ImageRegistryPrivateConfiguration
 	ImageRegistryPrivateConfigurationUser = ImageRegistryPrivateConfiguration + "-user"
 
+	// ImageRegistryOperatorNamespace is the namespace containing the registry operator
+	// and the registry itself
 	ImageRegistryOperatorNamespace = "openshift-image-registry"
+
+	// Status Conditions
+
+	// OperatorStatusTypeRemoved denotes that the image-registry instance has been
+	// removed
+	// TODO: does this serve any purpose?  As soon as it's removed we'll bootstrap
+	// a new one.
+	OperatorStatusTypeRemoved = "Removed"
```

Comment:
I vote +1 to removing this status.

Reply:
@coreydaley This status is needed at least for tests as there is no other way to know whether all resources have been deleted or not.